src/cascadia/TerminalApp/CommandPalette.idl

```diff
@@ -15,6 +15,7 @@ namespace TerminalApp
 
         void ToggleVisibility();
         void SetDispatch(ShortcutActionDispatch dispatch);
+        void SetKeyBindings(Microsoft.Terminal.Settings.IKeyBindings keyBindings);
```

Comment:
if the root key binding handler ran through Page instead of _termcontrol_ we probably wouldn't need to have it on the command palette

Reply:
I'm confident there's a dupe for this, but I forgot its name

Reply:
We do. Somewhere!